mlflow/models/evaluation/default_evaluator.py

```diff
@@ -41,20 +42,30 @@
 _DEFAULT_SAMPLE_ROWS_FOR_SHAP = 2000
 
 
-def _infer_model_type_by_labels(labels):
+def _is_categorical_values(labels):
```

Comment:
Can we call this function `_is_categorical_feature`?

```suggestion
def _is_categorical_feature(values):
```

Reply:
This method is also called from `_infer_model_type_by_labels`, used to check labels type.

Reply:
ok, can we just call `_is_categorical`?